### DISCUSSION TITLE
Building the code locally does not work due to authenticated NuGet feed

### DISCUSSION THREAD

**Discussion by rodyvansambeek** [Azure/data-api-builder#1499] [Q&A] [answered]
Hi, 

Today I tried to run the code locally to start working on #692 but while following the [Contributing docs](/CONTRIBUTING.md) I get a list of NuGet errors which are failed to download the packages:

```powershell
C:\Program Files\dotnet\sdk\6.0.100\NuGet.targets(130,5): error : Failed to download package 
'Microsoft.NETCore.App.Runtime.osx-x64.6.0.0' from 
'https://pkgs.dev.azure.com/sqldab/fcb212b3-b288-4c9e-b55a-5842a268b16d/_packaging/3d591251-bc27-4113-859d-0a355c21e037/nuget/v3/flat2/microsoft.netcore.app.runtime.osx-x64/6.0.0/microsoft.netcore.app.runtime.osx-x64.6.0.0.nupkg'.
[C:\Git\data-api-builder\src\Azure.DataApiBuilder.sln]

... (much more)
```

Looking at the Nuget feed in the Nuget.config file, I can see that this feeds needs an authenticated user to be able to download the packages. The authenticated user should be a member of the Microsoft tenant, which I am not. 
Is there something I can do for this to work around this? Can't seem to be able to build the project locally otherwise.

**Comment by Aniruddh25** [comment, marked as answer]
We are sorry, we will investigate, in the meantime as a workaround could you please try using nuget.org as the only package source: 
`https://api.nuget.org/v3/index.json`

**Reply by erinlcs** [reply]
This appears to still be an issue as of today, and this workaround is still valid.

**Comment by sezal98** [comment]
This is resolved . Please check and reopen in case of any issues
